src/schemas/sign-request.generated.ts

```diff
@@ -13,7 +13,12 @@ export interface SignRequest extends schemas.SignRequestCreateRequest {
 	/**
 	 * Array of signers for the sign request
 	 */
-	signers: schemas.SignRequestSigner[];
+	signers?: schemas.SignRequestSigner[];
+	/**
+	 * Force a specific color for the signature (blue, black, or red).
+	 * Example: blue
+	 */
+	signature_color?: string;
```

Comment:
I suppose that the changes to the Sign API that were made since the release aren't reflected in other SDKs either...